python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/cli_invocation.py

```diff
@@ -54,6 +54,12 @@ def wait(
         self, timeout: Optional[float] = None
     ) -> Iterator[Union[AssetCheckEvaluation, AssetCheckResult, AssetMaterialization, Output]]:
         run = self.run_handler.wait(timeout=timeout)
+
+        # Write dbt Cloud run logs to stdout
+        logs = self.run_handler.get_run_logs()
+        if logs:
+            print(logs, flush=True)  # noqa: T201
```

Comment:
should use `sys.stdout.write()` instead of print (that'll get rid of that noqa for you as well)